web/src/settings_linkifiers.ts

```diff
@@ -11,6 +11,7 @@ import * as channel from "./channel.ts";
 import * as confirm_dialog from "./confirm_dialog.ts";
 import * as dialog_widget from "./dialog_widget.ts";
 import {$t_html} from "./i18n.ts";
+import {python_to_js_linkifier} from "./linkifiers.ts";
```

Comment:
Please import this with `import *`, as that's our pattern for all but a handful of functions/modules.